src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -32,22 +32,30 @@ public void promptNextPlayer(){
     public void playGame(){
         Scanner keyboardScanner = new Scanner(System.in);
 
-        while (board.getWinner() == null){
-            board.printBoard();
+        while (board.getWinner() == null && board.isTie() == false){
```

Comment:
I'd usually prefer `!board.isTie()` to `board.isTie() == false`. If it matches the style here though then it's fine.